prow/pod-utils/gcs/target.go

```diff
@@ -56,6 +56,20 @@ func AliasForSpec(spec *pjutil.JobSpec) string {
 	return ""
 }
 
+// LatestBuildForSpec determines the GCS path for storing the latest
+// build id for a job.
+func LatestBuildForSpec(spec *pjutil.JobSpec) string {
+	switch spec.Type {
+	case kube.PeriodicJob, kube.PostsubmitJob:
+		return path.Join("logs", spec.Job, "latest-build.txt")
+	case kube.PresubmitJob, kube.BatchJob:
+		return path.Join("pr-logs", "directory", spec.Job, "latest-build.txt")
+	default:
+		logrus.Fatalf("unknown job spec type: %v", spec.Type)
```

Comment:
this should be logrus.Errorf, unless you want to crash the whole process here

Reply:
Fixed

Reply:
Although the rest of the helpers use Fatalf for the same cause, ie. can't figure out the pj type so `initupload` will crash anyway.

Reply:
Since I am going to use this helper in tot I think it's better to avoid crashing.